packages/parcel-resolver-storybook/StorybookResolver.ts

```diff
@@ -53,43 +50,5 @@ module.exports = new Resolver({
         priority: 'sync',
       };
     }
-
-    // Workaround for pkg#exports support
```

Comment:
So I assume this means that you have to enable package exports manually in your package.json now? we might need to add that to the docs (readme)

Reply:
done